model/logs/events.yaml

```diff
@@ -9,24 +9,10 @@ groups:
         type: string
         requirement_level: required
         brief: >
-          The name identifies the event.
-        examples: ['click', 'exception']
-      - id: domain
-        brief: >
-          The domain identifies the business context for the events.
-        type:
-          allow_custom_values: true
-          members:
-            - id: browser
-              value: 'browser'
-              brief: 'Events from browser apps'
-            - id: device
-              value: 'device'
-              brief: 'Events from mobile apps'
-            - id: k8s
-              value: 'k8s'
-              brief: 'Events from Kubernetes'
-        requirement_level: required
+          Identifies the class / type of event.
         note: >
-          Events across different domains may have same `event.name`, yet be
-          unrelated events.
+          Event names are subject to the same rules as [attribute names](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md).
```

Comment:
This should not point to main. This repo targets a specific spec release, which is mentioned in here: https://github.com/open-telemetry/semantic-conventions#opentelemetry-semantic-conventions

> This repository is currently using [this specification version](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.26.0).

And in CONTRIBUTION there's a guide for maintainers in how to update the spec version in the entire repo: https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#updating-the-referenced-specification-version

Reply:
Thanks @joaopgrassi for pointing this out, this is the part I was looking for.